### ISSUE TITLE
maven settings

### ISSUE THREAD

**Issue by danish-monu** [borysn/spring-boot-angular2#22]
is it possible that you package the whole thing as a maven project?

**Comment by borysn** [comment]
@danish-monu, yes very possible, gradle actually uses maven central. however, for this specific project I made a decision to use gradle, so I won't be supporting a maven config. feel free to fork and write up your own! 

I think the early commits for the project used maven, you could possibly shift through and then revert to see the config. 

I don't have the time to support that, but it may be an option. 

writing up a maven config is pretty simple though. the spring doc has a lot of examples you can find.
